### PR TITLE
Add 5 Canadian Mobile Carriers

### PR DESCRIPTION
**Pull request by daylenca**
Adds Bell, Freedom Mobile, SaskTel, Shaw and Virgin Plus.

### CODE REVIEW

File: entries/b/bell.ca.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Bell": {
+    "domain": "bell.ca",
+    "contact": {
+      "facebook": "BellCanada",
+      "twitter": "Bell"
```
Comment (Carlgo11) [inline entries/b/bell.ca.json]: Support accounts are preferred as they are actively monitored by the company.
```suggestion
      "twitter": "Bell_Support"
```

File: entries/f/freedommobile.ca.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "Freedom Mobile": {
```
Comment (Carlgo11) [inline entries/f/freedommobile.ca.json:2]: I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

File: entries/s/sasktel.com.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "SaskTel": {
+    "domain": "sasktel.com",
+    "contact": {
+      "facebook": "SaskTel",
+      "twitter": "SaskTel"
```
Comment (Carlgo11) [inline entries/s/sasktel.com.json]:
```suggestion
      "twitter": "SaskTelSupport"
```

File: img/b/bell.ca.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...ve" viewBox="0 0 269.553 154.412"><g fill="#0067A4"><path d="M200.294 89.145c-1.415-6.842-4.122-12.499-8.027-17.649-8.002-10.561-20.964-17.236-38.508-17.236-27.274 0-44.043 18.521-49.009 38.044-4.46-13.766-16.396-23.053-29.004-26.977 9.49-6.918 14.458-16.814 14.458-28.111 0-10.486-4.365-20.031-11.672-26.243C67.812 1.86 53.2 0 35.195 0H0v151.975c9.972 0 20.222-.044 30.214 0 8.89.041 17.572-.012 25.668-.336 25.626-1.027 43.166-13.926 48.78-33.296 1.928 8.612 5.636 15.633 10.478 20.961 9.868 10.862 24.175 15.108 38.583 15.108 17.74 0 30.566-5.145 39.328-13.029l-12.541-14.74c-6.311 4.758-14.422 7.563-23.531 7.563-17.865-.056-25.743-11.36-26.449-22.229h69.763v39.998h26.131V.759h-26.131v88.386zM26.851 20.532l7.261-.004c7.012 0 12.55.754 16.538 2.302 7.874 3.055 11.617 9.316 11.617 17.137.009 6.613-2.521 17.005-16.432 19.846-2.327.475-5.679.738-10.415.738h-8.569V20.532zm29.541 109.2c-3.271.588-7.271.719-10.52.719H26.849l.001-49.58h17.021c5.372 0 11.451.604 14.934 1.516 11.941 3.133 18.801 11.695 18.801 23.033 0 11.74-6.317 21.628-21.214 24.312zm74.619-36.65c2.303-11.07 10.486-19.038 22.546-19.038 12.073 0 20.104 7.139 22.185 19.038h-44.731zM243.868.759h25.686v151.215h-25.686z"/></g></svg>
```
Comment (Carlgo11) [inline img/b/bell.ca.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 269.6 154.4"><path fill="#0067A4" d="M200.3 89.1a42.8 42.8 0 0 0-8-17.6 46 46 0 0 0-38.5-17.2c-27.3 0-44 18.5-49 38a43.4 43.4 0 0 0-29-27 34 34 0 0 0 14.4-28c0-10.6-4.4-20.1-11.7-26.3C67.8 1.9 53.2 0 35.2 0H0v152h30.2c8.9 0 17.6 0 25.7-.4 25.6-1 43.1-13.9 48.8-33.3a45.2 45.2 0 0 0 10.4 21 50.8 50.8 0 0 0 38.6 15.1c17.8 0 30.6-5.1 39.4-13l-12.6-14.8a38.8 38.8 0 0 1-23.5 7.6c-17.9 0-25.8-11.4-26.5-22.2h69.8v40h26.1V.8h-26.1V89zM26.9 20.5H34c7 0 12.6.8 16.6 2.3 7.8 3 11.6 9.3 11.6 17.2 0 6.6-2.6 17-16.5 19.8a55 55 0 0 1-10.4.8H27v-40zm29.5 109.2c-3.3.6-7.3.8-10.5.8h-19V80.9h17a68 68 0 0 1 15 1.5c11.8 3.1 18.7 11.7 18.7 23 0 11.8-6.3 21.6-21.2 24.3zM131 93.1c2.3-11 10.5-19 22.6-19 12 0 20 7 22.1 19H131zM244 .8h25.7V152H244z"/></svg>
```

File: img/f/freedommobile.ca.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 36"><defs><path id="a" d="M.323.327h16.151V22.42H.323z"/></defs><g fill="none" fill-rule="evenodd"><g transform="translate(60 .058)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path fill="#2F7DC0" d="M2.196.327 16.474 22.42S5.02 20.877 1.683 14.627c-3.336-6.25.513-14.3.513-14.3" mask="url(#b)"/></g><path fill="#2F7DC0" d="m73.087 1.18 3.387 21.297s-8.079-4.951-8.51-10.744c-.43-5.791 5.123-10.552 5.123-10.552"/><path fill="#00558C" d="M67.964 11.734c.431 5.792 8.51 10.743 8.51 10.743l-8.39-12.982a9.06 9.06 0 0 0-.12 2.239"/></g></svg>
```
Comment (Carlgo11) [inline img/f/freedommobile.ca.svg]: I think you tried using parts of an SVG without changing the viewBox (width/height) of the SVG. Here's one with with a changed viewBox:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16.15 22.09"><path fill="#2f7dc0" d="M1.87 0s-3.85 8.05-.5 14.3c3.33 6.25 14.78 7.8 14.78 7.8L12.77.8s-4.18 3.58-5 8.31z"/><path fill="#00558c" d="M7.64 11.35c.43 5.8 8.51 10.74 8.51 10.74L7.76 9.11a9.06 9.06 0 0 0-.12 2.24"/></svg>
```

File: img/s/sasktel.com.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 140 24"><g fill="#0074B3" fill-rule="evenodd"><path d="M140 .505h-30.468v2.779h11.752V1.953H140zM139.245 4.547h-30.468v2.779h13.084V5.997h17.384zM138.489 8.589h-30.467v2.779h14.415v-1.332h16.052zM137.986 12.379h-30.468v2.779h15.707v-1.33h14.761zM137.23 16.421h-30.467V19.2h17.042v-1.33h13.425zM136.475 20.463h-30.468v2.779h18.412v-1.329h12.056z"/></g></svg>
```
Comment (Carlgo11) [inline img/s/sasktel.com.svg]: Another SVG resize:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 34 22.74"><path fill="#0074b3" d="M34 0H3.53v2.78h11.75V1.45H34Zm-.75 4.05H2.78v2.78h13.08V5.5h17.39Zm-.76 4.04H2.02v2.78h14.42V9.54h16.05zm-.5 3.79H1.52v2.78h15.7v-1.33H32Zm-.76 4.04H.76v2.78H17.8v-1.33h13.43zm-.75 4.04H0v2.78h18.4v-1.33h12.06z"/></svg>
```

File: img/s/shaw.ca.svg [hunk 7]
```diff
@@ -0,0 +1 @@
...965c-1.97-.578-11.135-3.045-21.075-3.045-13.891 0-20.724 5.625-20.724 14.006 0 8.473 5.976 15.161 23.738 15.161 7.359 0 14.057-1.31 18.062-2.889zm-21.22 96.75c-12.57 0-26.163-2.086-35.926-5.058-3.695-1.156-6.131-3.489-6.131-8.03V149.26c-.052-3.344 2.147-6.089 8.143-4.727 7.029 1.62 19.682 4.273 31.448 4.273 15.45 0 23.687-7.204 23.687-18.289v-4.139c-7.596 1.786-15.038 2.942-24.027 2.942-32.16 0-52.163-14.51-52.163-40.727 0-22.499 17.04-38.662 51.14-38.662h.352c2.95.021 3.984.031 6.46.031 18.403 0 39.685 5.491 45.692 7.142 2.343.64 6.698 1.858 6.698 7.07v60.192c.072 19.796-3.478 30.767-12.055 39.158-8.257 8.081-22.49 13.19-43.317 13.19m237.959-3.7h-21.086c-4.293 0-6.471-2.271-6.471-6.471V96.639c0-7.803-4.552-15.688-17.515-15.688-8.587 0-16.627 3.922-16.627 11.59v74.002c0 4.335-2.272 6.471-6.451 6.471h-21.086c-4.283 0-6.45-2.271-6.45-6.471V92.541c0-8.257-8.36-11.59-17.35-11.59-12.86 0-16.72 7.575-16.72 15.688v69.904c0 4.335-2.25 6.471-6.44 6.471h-21.086c-4.283 0-6.461-2.271-6.461-6.471V93.945c0-27.423 19.817-42.492 48.54-42.492 18.175 0 30.26 6.616 36.485 11.188 6.337-4.603 17.72-11.322 35.029-11.322 28.744 0 50.15 11.9 50.15 42.626v72.598c0 4.335-2.282 6.471-6.462 6.471"/></g></svg>
```
Comment (Carlgo11) [inline img/s/shaw.ca.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 896 307"><path fill="#0488c1" d="M274 85c-12 0-30 4-44 13V38c0-6-3-9-9-9h-31c-5 0-9 3-9 9v203c0 6 3 8 9 8h31c6 0 9-3 9-8v-99c0-11 14-19 30-19 9 0 17 3 21 7s6 9 6 17v94c0 6 3 8 8 8h31c6 0 9-3 9-8v-94c0-43-26-62-61-62M95 117c-36-11-41-18-41-29 0-12 11-18 30-18 18 0 43 6 51 8 6 1 9-3 9-7V49c0-4-1-9-8-11-15-5-34-8-57-8C42 31 4 46 4 91c0 33 20 52 65 66 31 10 42 19 42 32 0 14-12 21-38 21-28 0-53-11-63-13-7-2-10 1-10 6v25c0 7 5 10 10 11 1 1 28 12 63 12 59 0 87-24 87-64 0-44-28-58-65-70M804 3c-2-1-5-3-8-3-5 0-7 3-7 8v32c0 4 1 7 4 9l2 2c35 23 56 58 56 102 0 45-21 80-56 103a83 83 0 0 0-2 2c-3 2-4 4-4 8v33c0 5 2 8 7 8 3 0 6-2 8-3a169 169 0 0 0 0-301M457 214c-2 1-15 4-28 4-18 0-27-7-27-19 0-11 8-20 31-20 10 0 19 2 24 4zM429 85c-17 0-35 3-48 7-5 1-8 4-8 10v20c0 4 3 8 11 6 9-2 26-6 42-6 20 0 31 10 31 25v5c-10-2-20-4-32-4-43 0-69 20-69 55 0 30 22 51 68 51h9c25 0 53-7 61-9 3-1 9-3 9-10v-80c0-27-5-41-16-52s-30-18-58-18m317 5h-28c-6 0-8 3-8 9v93c0 10-7 21-24 21-11 0-22-5-22-16V99c0-6-3-9-9-9h-28c-5 0-8 3-8 9v98c0 11-11 16-23 16-18 0-23-10-23-21V99c0-6-3-9-8-9h-28c-6 0-9 3-9 9v96c0 37 26 57 65 57 24 0 40-9 48-15 9 6 24 15 47 15 38 0 67-16 67-57V99c0-6-3-9-9-9"/></svg>
```

File: img/v/virginplus.ca.svg [hunk 8]
```diff
@@ -0,0 +1 @@
...6.547 1.399 10.235-6.794 13.513-13.725 10.534-22.272-1.186-3.401-3.002-4.19-7.437-3.232m56 33.574c-4.4 1.962-11.15 4.24-15 5.063-7.933 1.695-14.441 5.081-16.858 8.77-3.879 5.919-4.723 10.985-6.177 37.076-.307 5.5-.943 12.475-1.413 15.5-.471 3.025-.908 9.55-.971 14.5-.063 4.95-.384 10.915-.712 13.256-1.689 12.025 2.591 20.999 9.481 19.881 8.728-1.416 17.448-8.794 19.229-16.268.507-2.128 1.371-12.644 1.92-23.369.55-10.725 1.418-21.3 1.93-23.5 2.771-11.906 3.678-14.403 6.803-18.734 1.878-2.604 4.845-5.742 6.592-6.972 1.747-1.23 4.296-3.262 5.666-4.516 1.369-1.253 2.795-2.278 3.168-2.278 1.448 0 4.903-6.993 5.022-10.166.232-6.161-4.357-11.875-9.495-11.822-.652.007-4.785 1.617-9.185 3.579m-76.249 32.438c-9.459 5.425-12.925 13.274-14.34 32.471-.526 7.15-1.422 17.725-1.989 23.5-.567 5.775-1.28 19.5-1.583 30.5-.537 19.465-.494 20.06 1.594 22.25 3.027 3.176 7.68 2.917 13.121-.729 11.868-7.952 11.389-7.47 11.544-11.617.393-10.538 2.51-34.363 3.368-37.904.533-2.2 1.249-8.05 1.591-13 .342-4.95 1.268-12.629 2.058-17.065 1.866-10.476 1.374-25.276-.957-28.835-2.39-3.646-7.569-3.492-14.407.429M337.169 651.75c94.968.141 250.668.141 346 0s17.631-.256-172.669-.256-268.299.115-173.331.256"/></g></svg>
```
Comment (Carlgo11) [inline img/v/virginplus.ca.svg]: This SVG looks like it's been run through a raster to vector image ML program and I have yet to see any decent results from such programs.
As there doesn't seem to be a proper SVG for Virgin Plus please replace it with a PNG. A+ for searching for a SVG file tho! :+1:

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/f/freedommobile.ca.json (hunk 2 above)
Comment (daylenca) [reply entries/f/freedommobile.ca.json:2]: Freedom's 2FA doesn't have any sign-up process or settings. It's a mandatory feature that is tied to the mobile line(s) purchased through the carrier and the billing e-mail. Here is the login process though:
image
image
image

File: img/v/virginplus.ca.svg (hunk 8 above)
Comment (daylenca) [reply img/v/virginplus.ca.svg]: I have updated the Virgin Plus logo with a PNG, but the file is still 3.7 kB, even after running it through TinyPNG.

**Review by Carlgo11 — APPROVED** [review]
Thanks for your contribution!
Apologies for the delay in reviewing the changes. Had to go away for a couple of days.

**Pull request merged**